book/views.py

```diff
@@ -1,3 +1,32 @@
-from django.shortcuts import render
+import json
+from datetime         import timedelta, date
 
-# Create your views here.
+from django.views     import View
+from django.http      import JsonResponse
+
+from .models          import Book
+
+
+class CommingSoonView(View):
+    def get (self, request):
+        day    = request.GET.get('day')
+        limit  = request.GET.get('limit', '20')
+
+        today            = date.today()
+        next_publication = today + timedelta(days=int(day))
+        min_day          = today + timedelta(days=1)
+        max_day          = today + timedelta(days=5)
+
+        books = Book.objects.filter(
+            publication_date__range=
+            [min_day, next_publication]).order_by('publication_date')[:int(limit)]
+
+        book_list = [{
+            "title"  : book.title,
+            "image"  : book.image_url,
+            "author" : book.author,
+            "date"   : (book.publication_date - today).days
+            if min_day <= book.publication_date <= max_day
+            else book.publication_date.strftime('%m월%d')
```

Comment:
```suggestion
            "date"   : (book.publication_date - today).days
                if min_day <= book.publication_date <= max_day
                else book.publication_date.strftime('%m월%d')
```
시각적으로 코드의 계층을 분리해주시는 것이 좋을 것 같습니다.

Reply:
@soheon-lee 
수정했습니다. 감사합니다.